Makefile

```diff
@@ -299,9 +311,11 @@ docopen: out/doc/api/all.html
 docclean:
 	-rm -rf out/doc
 
-run-ci:
+build-ci:
```

Comment:
Building the addons should happen only in the test machine that tests addons. We could add it here and transfer all the output to the test machines, but bandwidth is low so this would be quite slow. The Raspberries that test addons handle it well (compile + test addons is faster than all the other tests divided by 7).